packages/services/api/src/context.ts

```diff
@@ -5,7 +5,7 @@ export interface RegistryContext {
   requestId: string;
   user: any;
   headers: Record<string, string | string[] | undefined>;
-  abortSignal: AbortSignal;
+  request: Request;
```

Comment:
We now have `req` and `request`. Do you think we should merge these two?

Reply:
They were in the context before, I only added type for `request: Request;`, so I can re-use it for the abort signal

Reply:
I think we should replace all usages of `req` with `request` where possible